department-of-veterans-affairs/caseflow
# Generate CO Hearing Day schedule as part of RO Build Schedule

Resolves #8844 

### Description
Added logic to add CO Hearing Days as part of building the RO schedule.

### Acceptance Criteria

### Testing Plan
1. Go to Build Schedule page
2. Click on Upload Files
3. Select Apr 1, 2019 to Sept 30, 2019
4. Upload attached template
5. Confirm CO hearing days are created following the parameters stated in the issue ticket.


[ROAssignmentsAprSept2019_v1.xlsx](https://github.com/department-of-veterans-affairs/caseflow/files/2865711/ROAssignmentsAprSept2019_v1.xlsx)
